technical_architecture/performances_guide/php7_hhvm.rst

```diff
@@ -8,7 +8,7 @@ We continued our effort regarding Akeneo PIM PHP7 support.
 
 We're happy to announce that PHP7 is now supported and production ready for both CLI and Web.
 
-This modification introduces a new constraint, the minimum version of PHP for Akeneo PIM is now PHP 7.1.
+This modification introduces a new constraint, the minimum version of PHP for Akeneo PIM 2.x is now PHP 7.1.
```

Comment:
and 7.2 for Akeneo 3.x ?

Reply:
This page is legacy, I think it should disappear from master, so I didn't touch it for now.

Reply:
yes, we should remove this page